app/main.js

```diff
@@ -33,23 +45,27 @@ const appEnter = (nextState, replace, callback) => {
       replace('/');
     callback();
   }));
-}
+};
+
 const newPlanEnter = (nextState, replace) => {
```

Comment:
In my testing, this async() action is occurring post-enter, thereby throwing an error in the code related to this.props.exercises. Or in other words, because exercises are loading after code in newplan.js is using those exercises, we're getting errors. Needs debug.